test/README.md

```diff
@@ -41,3 +41,52 @@ Check the usage of the tests script:
 - [Go](https://github.com/microsoft/vscode-go) extension
 - [Go Documentation](https://github.com/msyrus/vscode-go-doc)
 
+### How to Debug in VS Code
+
+- Run the command (press *F1*), type "Go: Install/Update Tools", select dlv, press Ok to install/update delve.
+- Run the command (press *F1*), type "Debug: Open launch.json" with *attach* mode:
+
+```json
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch",
+            "type": "go",
+            "request": "attach",
+            "remotePath": "${workspaceFolder}",
+            "mode": "remote",
+            "port": 2345,
+            "host": "127.0.0.1"
+        }
+    ]
+}
+```
+
+| If you didnt already have a launch.json file, this will create one with the below default configuration which can be used to debug the current package.
+
+- Install Delve go module:
+
+```sh
+go get -v github.com/go-delve/delve/cmd/dlv
+```
+
+- Start the Delve process to launch our tests:
+
+```sh
+dlv test --headless --listen=:2345 --log --api-version=2 -- -godog.tags="" features/my_feature.feature
+```
+
+The program will wait until you launch the Debug mode from VS Code (next step).
+
+| It must run in the /test folder
+| Any arguments must be passed after the "--" token, for example: "-- -godog.tags="" -tests.services-image-version=0.9.0-rc2"
+
+- Launch Debug from VS Code
+
+The launch icon is found at the right side of VS Code, open it and then run the Play icon at the top menu of the side bar. 
```

Comment:
I am not sure what are you referring to here, I launched debug session by selecting Run/Start Debugging from top menu.

Reply:
I will rewrite the sentence to make it clearer. Thanks!

Reply:
Updated